app/src/main/res/values/strings.xml

```diff
@@ -68,6 +69,83 @@
         <item>CH2</item>
     </string-array>
 
+    <string-array name="multimeter_knob_states_title">
+        <item>Channel 1</item>
+        <item>CAP</item>
```

Comment:
what's this?

Reply:
It reads the voltage at CAP pin. Not sure what to put as full form there.